packages/spineClient/src/live-spine-client.ts

```diff
@@ -116,4 +116,13 @@ export class LiveSpineClient implements SpineClient {
       return serviceHealthCheck(process.env.healthCheckUrl, logger, new Agent())
     }
   }
+
+  async isCertificateConfigured(): Promise<boolean> {
```

Comment:
This does not need to be async - it can just be a normal function that returns a boolean. There will be other changes related to this as well - eg no await when calling it

Reply:
The method has been amended